integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/predicate/MatchSearchPredicateIT.java

```diff
@@ -864,6 +864,72 @@ public void analyzerOverride_notExistingName() {
 		;
 	}
 
+	@Test
+	public void analyzerIgnore() {
```

Comment:
This got me thinking: even if we don't provide the ability to override normalizers, ignoring analysis could make sense for normalized fields too.

Currently you simply ignore the `ignoreAnalyzer` flag if the field is normalized instead of analyzed. But you could skip normalization when the flag is enabled. That would require to rename the option, though. `skipAnalysis()` may make sense for both analyzers and normalizers?
We would need an additional test, just like this one but on normalized fields.

The option would make sense for simple query string predicates too, and for consistency I guess we could support it on phrase predicates (or just remove the option entirely for phrase predicates as we discussed on zulip: just see what's easier for you).